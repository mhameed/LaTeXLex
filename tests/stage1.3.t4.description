author: ehc
date: 09/04/01
description: check that things such as x^{\rm yes} are dealt with correctly