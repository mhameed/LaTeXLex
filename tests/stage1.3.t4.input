Here is some silly maths
$$
{\rm C}(x)^{\rm yes}
$$
and some more ${\cal L}_{\bf no}$
