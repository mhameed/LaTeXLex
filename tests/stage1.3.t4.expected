Here is some silly maths
$$
C(x)^{yes}
$$
and some more $\calL_{no}$
